ChatSecure/Classes/Controllers/XMPP/Storage/RoomStorage.swift

```diff
@@ -103,6 +103,10 @@ import YapDatabase
             }
             if message.buddyUniqueId == nil, let senderJidString = message.senderJID, let senderJid = XMPPJID(string: senderJidString), let roomJid = room.roomJID, let occupant = OTRXMPPRoomOccupant.occupant(jid: senderJid, realJID: senderJid, roomJID: roomJid, accountId: account.uniqueId, createIfNeeded: false, transaction: transaction), let buddy = occupant.buddy(with: transaction) {
```

Comment:
@N-Pex Minor nitpick (not blocking this merge) but in the future please insert newlines between the if conditions when they don't fit on a single line. Xcode will automatically line them up if you hit enter after the ", "

Reply:
Ok!